src/Widgets/FormInput/FormInput.tsx

```diff
@@ -83,7 +85,27 @@ export const FormInput = forwardRef<FormSubmit, FormInputProps>((props, ref):JSX
                         control={control}
                         rules={{
                         required: field.required,
-                        pattern: field.validationRegex
+                        pattern: field.validationRegex,
+                        validate: async (value, formValues) => {
+                            var responseStatus = false;
+                            // displayName field does not have its own InputType. Because of this, we have to perform validation for that field specifically
+                            if (fieldValueIsString(field.type, value) && field.field == "displayName" && field.unique) {
```

Comment:
Can we be more generic?
_Currently just have `email` and `displayName`, but it'd be nice to only have to update the config if this changes._

`if(props.validateUniqueFields && field.unique && value.match(field.validationRegex)) => then call service`

- Seems complicated enough to maybe move to a utility function
- Doesn't necessarily have to be a string.  The URL query param will make a string anyways.  And if we later have a unique field that needs to be evaluated.  The server can handle parsing.